source/navier_stokes_solver.py

```diff
@@ -555,3 +579,347 @@ def solve(self):
         self._newton_problem.F(residual_vector, self._solution.vector())
         residual = residual_vector.norm("l2")
         assert residual <= self._tol, "Newton iteration did not converge."
+
+
+class InstationaryNavierStokesSolver(NavierStokesSolverBase):
+    _sub_space_association = {0: "velocity", 1: "pressure"}
+    _field_association = {value: key for key, value in _sub_space_association.items()}
+
+    def __init__(self, mesh, boundary_markers, time_stepping, tol, max_iter):
```

Comment:
Why didn't set some default parameters here, like ```StationaryNavierStokesSolver```
```python
class StationaryNavierStokesSolver(NavierStokesSolverBase):   
      def __init__(self, mesh, boundary_markers, tol=1e-10, maxiter=50, tol_picard=1e-2, maxiter_picard=10):
```

Reply:
In fact, we could use default values for the parameters `tol` and `max_iter`.

Reply:
I added default values.